pkg/ccl/streamingccl/logical/logical_replication_writer_processor.go

```diff
@@ -479,35 +389,10 @@ func (lrw *logicalReplicationWriterProcessor) handleEvent(event streamingccl.Eve
 	default:
 		return errors.Newf("unknown streaming event type %v", event.Type())
 	}
-
-	if lrw.logBufferEvery.ShouldLog() {
-		log.Infof(lrw.Ctx(), "current KV batch size %d (%d items)", lrw.buffer.curKVBatchSize, len(lrw.buffer.curKVBatch))
-	}
-
-	shouldFlush, mustFlush := lrw.buffer.shouldFlushOnKVSize(lrw.Ctx(), sv)
-	if mustFlush {
-		if err := lrw.flush(flushOnSize); err != nil {
-			return err
-		}
-	} else if shouldFlush {
-		if err := lrw.maybeFlush(flushOnSize); err != nil {
-			return err
-		}
-	}
 	return nil
 }
 
-func (lrw *logicalReplicationWriterProcessor) bufferKVs(kvs []roachpb.KeyValue) error {
-	if kvs == nil {
-		return errors.New("kv event expected to have kv")
-	}
-	for _, kv := range kvs {
-		lrw.buffer.addKV(kv)
-	}
-	return nil
-}
-
-func (lrw *logicalReplicationWriterProcessor) bufferCheckpoint(event streamingccl.Event) error {
+func (lrw *logicalReplicationWriterProcessor) checkpoint(event streamingccl.Event) error {
```

Comment:
Should this take a ctx and select on ctx.Done() during the send? Or perhaps on the stopCh?